db/migrate/20150625164705_make_merged_at_datetime.rb

```diff
@@ -0,0 +1,6 @@
+class MakeMergedAtDatetime < ActiveRecord::Migration
+  def change
+    remove_column :hubstats_pull_requests, :merged_at
+    add_column :hubstats_pull_requests, :merged_at, :datetime
```

Comment:
I didn't test that, but I'm pretty sure that no other logic needs to be done because there isn't anything in the pull request model that does anything for the created_at, updated_at, or closed_at. I'm pretty sure it'll just automatically convert all pull requests.